arviz/plots/__init__.py

```diff
@@ -1,15 +1,15 @@
 """Plotting functions."""
-from .autocorrplot import autocorrplot
-from .compareplot import compareplot
-from .densityplot import densityplot
-from .energyplot import energyplot
-from .forestplot import forestplot
-from .kdeplot import kdeplot, _fast_kde, _fast_kde_2d
-from .parallelplot import parallelplot
-from .posteriorplot import posteriorplot
-from .traceplot import traceplot
-from .pairplot import pairplot
-from .jointplot import jointplot
-from .khatplot import khatplot
-from .ppcplot import ppcplot
-from .violintraceplot import violintraceplot
+from .autocorrplot import autocorrplot as plot_autocorr
+from .compareplot import compareplot as plot_compare
+from .densityplot import densityplot as plot_density
+from .energyplot import energyplot as plot_energy
+from .forestplot import forestplot as plot_forest
+from .kdeplot import kdeplot as plot_kde, _fast_kde, _fast_kde_2d
```

Comment:
If were exposing `_fast_kde`, and `_fast_kde_2d` for users we should remove the underscore to make it obvious its meant for end users.

If we don't intent for users to use this method we should remove it from the import